applications/launchpad_v2/src/components/Box/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { StyledBox } from './styles'
+
+import { BoxProps } from './types'
+
+const Box = ({ children, gradient, border, style: inlineStyle }: BoxProps) => {
```

Comment:
NIT: JSDoc could be added here, but also the component name is self-explanatory, so can be skipped.

Reply:
added